vitrina/datasets/models.py

```diff
@@ -363,6 +363,22 @@ class Dataset(TranslatableModel):
         on_delete=models.PROTECT,
         verbose_name=_("Informacinės sistemos tipas"),
     )
+    information_system_publisher = models.ForeignKey(
+        Organization,
+        related_name="information_system_publisher",
+        on_delete=models.PROTECT,
+        null=True,
+        blank=False,
```

Comment:
Field cannot should be `blank=False`, because field is only required for "information_system" subclass. `blank=True` will make saving  non "information_system" subclass `Datasets` impossible via admin.

Just don't forget to make it required in form
```suggestion
        blank=True,
```

Reply:
These are additional fields for information system based on dcat ap lt. They are/should be "non functional"